app/presenters/crime_application_presenter.rb

```diff
@@ -36,4 +36,8 @@ def interim_date_stamp
   def date_stampable?
     CaseType.new(case_type.to_s).date_stampable?
   end
+
+  def draft
+    CrimeApplication.find_by(parent_id: id)
```

Comment:
Can we add an index on the parent_id column of crime_applications to ensure this query remains performant?

Unrelated to your changes, but I also think we should add a unique constraint to the index to mitigate the slight risk of consecutive requests rehydrating an application.